invenio_app_ils/ui/src/semantic-ui/site/elements/container.overrides

```diff
@@ -20,80 +35,137 @@
     }
 
     &.fs-content {
-    margin-top: @headerMenuHeight;
-    flex: 1;
-    justify-content: space-between;
-  }
+      margin-top: @headerMenuHeight;
+      flex: 1;
+      justify-content: space-between;
+    }
 
     &.document-tags {
       margin-top: 2em;
       margin-bottom: 2em;
     }
-  }
 
-  .breadcrumbs{
-    padding-bottom: 2em;
-  }
+    &.fs-headline-section {
```

Comment:
is `fs-headline` and `fs-headline-section` on the same level? If section is inside headline, it can also be nested in teh styles `headline { section {...} ... }`

Reply:
fixed